kotlin-js/src/jsMain/kotlin/js/typedarrays/Uint8Array.kt

```diff
@@ -18,5 +22,12 @@ open external class Uint8Array(
     constructor(elements: JsIterable<Byte>)
     constructor(elements: ReadonlyArray<Byte>)
 
+    inline fun toByteArray(): ByteArray =
+        Int8Array(buffer, byteOffset, length).asByteArray()
+
+    @OptIn(ExperimentalUnsignedTypes::class)
```

Comment:
Looks like candidate for common compiler flag

Reply:
@turansky done.